Symfony/CS/Config/Config.php

```diff
@@ -92,19 +89,14 @@ public function getFinder()
         return $this->finder;
     }
 
-    public function level($level)
-    {
-        $this->level = $level;
-
-        return $this;
-    }
-
-    public function getLevel()
-    {
-        return $this->level;
-    }
-
-    public function fixers($fixers)
+    /**
+     * Set fixers.
+     *
+     * @param FixerInterface[] $fixers
+     *
+     * @return $this
+     */
+    public function fixers(array $fixers)
```

Comment:
There is bigger problem with config, not related to this PR.
First, we should use consistent names.
Second, we use Config to allow user configure the tool, then resolve it, write back to config the resolved configuration and use it inside other classes.
Thus, we should have separate Config for user configuration and Config with resolved configuration. But that should be done as separate PR.